.github/workflows/ci-nightly.yml

```diff
@@ -1,6 +1,9 @@
 name: CI - FAST Publish
 
 on:
+  push:
+    branches:
+    
   pull_request:
```

Comment:
I would still prefer this run once nightly unless this would pose an issue.  How do you feel about this, @chrisdholt?   If we're all in agreement, then we should replace the `pull_request` event with a cron schedule and only for the master branch.

I do think that in our `ci-daily.yml` we can include the `yarn checkchange` to provide indication for contributors that more work is required before pushing to master to prepare for publishing.

Reply:
I think I'm tracking - I would expect publish to *only* run on a cron with a method of manually triggering the build in emergencies or urgent cases only. *Ideally* a manual deployment would be triggered via an ADO build agent behind permissions or something similar so that it doesn't require any individual to run it from their own CLI.

+ @EisenbergEffect if he has thoughts as well

Reply:
@chrisdholt, confirmed, we will be converting this to use Cron as soon as it's approved and as a final step before completing the PR.  We can put this in now if you want?

Reply:
@awentzel let's include as part of this PR if we can. 

Once this goes in, every current/existing PR which includes files under `/packages/*` *should* fail at `yarn checkchange`. Once this is ready, we should pause merging and publish. Immediately after the publish is complete, we should merge this in. That kicks off the official shift from publishing w/ Lerna to publishiing w/ Beachball.

Reply:
My original thought was to use NPM release channels.

* Canary Channel: Either triggered on every merge to master or by a nightly cron job. I don't have a strong opinion on which one. Should be at least once per day though.
* Beta Channel: Released once per week via a cron job.
* Stable Channel: Manually released by Chris, myself or other DRI at a planned time, typically in conjunction with messaging (e.g. blog post, tweets, conference, etc.).

Here's some information: https://github.com/danillouz/til-npm-release-channels
I think the biggest question was how to make this work with beachball.

Reply:
right now it is set to trigger 1 publish per night on cron job. Chris will know a little more.

Reply:
I think we'll want to ensure this meets Rob's expectations as well. We definitely can use a nightly publish. If we need to release those on the NPM cadence, I'm not sure this work accommodates that.

Reply:
I'd prefer to first get our publishing working and testing as an automated replacement of today's manual process.  Once that's in place and verified for a week or so we can most certainly discuss NPM channels.

- We may require contributing to beachball to get semver 2 meta data feature which to my knowledge has not been completed, such that we can properly tag the channel per previous GH Issues
- We may want to wait for GH/NPM Release capability to update.

In either case, that would be a good next step to discuss after we get the base automation in place.

Reply:
Sounds good. Just want to make sure the goal is clear.